src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseHybridFVM.cpp

```diff
@@ -460,19 +463,19 @@ void CompositionalMultiphaseHybridFVM::assembleFluxTerms( real64 const dt,
                                        faceGravCoef,
                                        mimFaceGravCoef,
                                        transMultiplier,
-                                       m_phaseDens.toNestedViewConst(),
-                                       m_dPhaseDens_dPres.toNestedViewConst(),
-                                       m_dPhaseDens_dComp.toNestedViewConst(),
-                                       m_phaseMassDens.toNestedViewConst(),
-                                       m_dPhaseMassDens_dPres.toNestedViewConst(),
-                                       m_dPhaseMassDens_dComp.toNestedViewConst(),
-                                       m_phaseMob.toNestedViewConst(),
-                                       m_dPhaseMob_dPres.toNestedViewConst(),
-                                       m_dPhaseMob_dCompDens.toNestedViewConst(),
-                                       m_dCompFrac_dCompDens.toNestedViewConst(),
-                                       m_phaseCompFrac.toNestedViewConst(),
-                                       m_dPhaseCompFrac_dPres.toNestedViewConst(),
-                                       m_dPhaseCompFrac_dComp.toNestedViewConst(),
+                                       compFlowAccessors.get( extrinsicMeshData::flow::phaseMobility{} ),
+                                       compFlowAccessors.get( extrinsicMeshData::flow::dPhaseMobility_dPressure{} ),
+                                       compFlowAccessors.get( extrinsicMeshData::flow::dPhaseMobility_dGlobalCompDensity{} ),
```

Comment:
This is the new way of extracting a specific accessor, hopefully pretty clean. We can also add `using namespace extrinsicMeshData` somewhere to make it even shorter.

Reply:
I only catching up with your modifications now, they look great! The getter for the accessor is now much better than in the previous version of the PR. Thanks! 

In addition to `using namespace`, we could also change `extrinsicMeshData` to something shorter as discussed previously (in a separate PR). I still think it is a good idea because `extrinsicMeshData` is quite long to type.

Reply:
@francoishamon @klevzoff Would it be possible to just pass in the accessors and do the extraction inside the kernel? It seems that the kernels define the accessor types, so they could have knowledge of that type?

Reply:
Isn't it what the crateAndLaunch does? Maybe that has not been created for the hybridFVM Kernel?

Reply:
You're both right, this should be done in the kernel class, which has not been created for hybrid solver yet because @francoishamon is going to address it in a future PR

Reply:
Yes, this needs to be done in a separate PR that will also take care of the aquifer boundary condition kernel, and the CFLFlux kernel. I did not want to change too many things in this PR before making sure that everyone was ok with the new style.